functions/src/strings.js

```diff
@@ -89,6 +89,11 @@ module.exports = {
       speech: 'State of FSM is {{state}}',
     },
 
+    globalError: {
+      speech: 'We got some problem with performing your request. Please rephrase it.',
```

Comment:
A few grammar issues here. Change speech to `We had some problems performing your request. Please rephrase it.`